doc/administrator/web/index.md

```diff
@@ -22,8 +22,8 @@ Prerequisites
 
 The following prerequisites are required for running the IRIDA web interfaces:
 
-* [Java](http://www.oracle.com/technetwork/java/index.html) 11 or higher.
-* A working servlet container supporting Servlet 3.1 ([Tomcat](https://tomcat.apache.org/), version 8 or higher, for example)
+* [Java](http://www.oracle.com/technetwork/java/index.html) 17 or higher.
+* A working servlet container supporting Servlet 3.1 ([Tomcat](https://tomcat.apache.org/), version 9 or higher, for example)
```

Comment:
Tomcat 10 is not supported, so up to Tomcat 9.

Reply:
Ah good catch. Updated in 7a5604de46298ba0f08714e23371877425ad1fb2